README.md

```diff
@@ -321,6 +321,28 @@ end
 For more information, see the [cucumber scenarios for mailer specs
 ](https://relishapp.com/rspec/rspec-rails/v/3-4/docs/mailer-specs).
 
+## Job specs
+
+Job specs default to residing in the `spec/jobs` folder. Tagging any context
```

Comment:
Inferring spec location is deprecated so these sentences are the wrong way around, although it does line up with the rest of the file may I suggest:

> Tagging a context with the metadata`:type => :job` treats it's examples as job specs. Typically these specs will live in `spec/jobs`.

You've already mentioned the config option in the actual cuke (🙌) so I think this will suffice